egs/reverb/s5/run.sh

```diff
@@ -33,305 +35,164 @@ fi
 . ./cmd.sh
 . ./path.sh
 
-stage=1
+stage=0
+nch_se=8
+# flag for turing on computation of dereverberation measures
+# please make sure that you or your institution have the license to report PESQ before turning on the flag
+compute_se=false
```

Comment:
I think it would be good to make it true. Why did you make it disable? Computational cost?

Reply:
Not just that. It is mainly because of the PESQ license issue. I have sent you a detailed email regarding this. To summarize, it is not clear if it is okay to report this result, the terms are obscure. So I added the comment "please make sure that you or your institution have the license to report PESQ before turning on the flag" like the reverb challenge.

Reply:
How about only make this flag available for PESQ? Then, all other metrics are computed by default.

Reply:
I am downloading and using the official scripts from REVERB challenge with very little patching. This might require significant patching, if you feel it is very important I can do that.